### DISCUSSION TITLE
Windows XP Horror Edition-like skin

### DISCUSSION THREAD

**Discussion by bfffggh** [Open-Shell/Open-Shell-Menu#1761] [General]
I want to make a project that converts any Windows ISO to a Windows XP Horror Edition ISO.
**I literally need an openshell skin for that.**

I'd like to get an Open-Shell skin file made of these two images:
[STARTMENU.zip](https://github.com/Open-Shell/Open-Shell-Menu/files/13325358/STARTMENU.zip)

### Area of issue

Start menu

### Alternatives I've considered

Making my own Open-shell skin
*that would be hell and agony*

### Additional context

The startmenu skin needs to be in the .skin file format.
It's supposed to ressemble Windows XP's start menu, and instead of including it with open-shell, Please just send me the finished file.
**My E-mail if necessary**: `sadlikkrzysztof16@gmail.com`
Hope the resources are useful to you!
Thank you!

**Comment by CTVCAM8** [comment]
Instead of your own hell and agony making a new skin, 
just use someone else's .skin from the old Classic Shell site:

http://classicshell.net/forum/viewtopic.php?f=17&t=8557


Then change the options and glass color to make it match.
For any changes that you can't find an option for, like "Main_separator lines with tapered ends",
then it's time to read up on how to modify a skin.

Note.
Classic/Open Shell can't do the original fat bottom of the XP start menu that has the Shutdown text on it.
It can only be faked.



![666](https://github.com/Open-Shell/Open-Shell-Menu/assets/44127216/9d744ee5-4e17-439a-94aa-75af445dd267)
